make/stanford.make

```diff
@@ -219,17 +227,12 @@ projects[context_useragent][subdir] = "contrib"
 projects[context_useragent][version] = "1.x-dev"
 projects[context_useragent][download][type] = "git"
 projects[context_useragent][download][url] = "http://git.drupal.org/project/context_useragent.git"
-projects[context_useragent][download][revision] = "8949c6d4ee7a462b49e1b08dfd7dc60f1fa7e8e8"
+projects[context_useragent][download][revision] = "3cd2c8c20cf9c075d5ccf8fe0bc2aa139823504c"
```

Comment:
I vaguely remember doing this, but can't find any documentation or discussion on in. The difference between those two commits was simply the patch we're removing, correct?